.pipelines/templates/ToolchainBuild.yml

```diff
@@ -1,4 +1,38 @@
+parameters:
+  - name: buildArtifactsFolder
+    type: string
+    default: "$(Build.ArtifactStagingDirectory)"
+
+  - name: coreRepoRoot
+    type: string
+    default: "$(Build.SourcesDirectory)"
+
 steps:
-  - script: |
-      echo "The used architecture is: $(uname -m)"
-    displayName: 'Use parameter'
+  - bash: sudo make -C "${{ parameters.coreRepoRoot }}/toolkit" "-j$(nproc)" toolchain QUICK_REBUILD=y
+    displayName: "Build toolchain"
+
+  - bash: |
+      failed_rpms_log="${{ parameters.coreRepoRoot }}/build/logs/toolchain/failures.txt"
+
+      if [[ -f "$failed_rpms_log" ]]; then
+          echo "List of RPMs that failed to build:" >&2
+          cat "$failed_rpms_log" >&2
+      else
+          echo "Build failed - no specific RPM" >&2
+      fi
+    condition: failed()
+    displayName: "Print failed RPMs"
+
+  - bash: |
+      published_artifacts_dir="${{ parameters.buildArtifactsFolder }}/ARTIFACTS"
+      mkdir -p "$published_artifacts_dir"
+      cp "${{ parameters.coreRepoRoot }}"/build/toolchain/toolchain_built_{,s}rpms_all.tar.gz "$published_artifacts_dir"
```

Comment:
Is the comma expected here? "{,s}"

Reply:
Yes, it's supposed to capture both `rpms_all.tar.gz` and `srpms_all.tar.gz`.